### PR TITLE
Fix: plot for ASSET documentation

### PR DESCRIPTION
**Pull request by Moritz-Alexander-Kern**
# plot for ASSET `plot_synchronous_events`

The plot created in the ASSET documentation 'viziphant.asset.plot_synchronous_events` is now displayed.

Check result  here: https://viziphant-experimental.readthedocs.io/en/fix-asset_plot/toctree/asset/viziphant.asset.plot_synchronous_events.html#viziphant.asset.plot_synchronous_events

---
Changes:
- added elephant[extras] to `requirements/requirements-docs.txt`
- added set environment variable to `doc/conf.py`

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('.'))
 
+# -- Setup environment variables for elephant --------------------------------
+# set the following environment variables to ensure execution of plots when
+# building docs. This is of importance in connection with elephant modules
+# using OpenCL (pyopencl) or CUDA (pycuda).
+
+os.environ["ELEPHANT_USE_OPENCL"] = "0"
```
Comment (mdenker) [inline doc/conf.py:22]: I think you also need
```
os.environ["ELEPHANT_USE_CUDA"] = "0"
```

File: requirements/requirements-docs.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ numpydoc>=1.1.0
 sphinx>=3.3.0
 sphinx-tabs>=1.1.13
 sphinxcontrib-bibtex==1.0.0
+elephant[extras]>=0.9.0 # needed for sphinx to create plots for GPFA and ASSET
```
Comment (mdenker) [inline requirements/requirements-docs.txt:5]: I would not comment this, I think it's clear we need the extras to be able to run any function in the elephant:
```suggestion
elephant[extras]>=0.9.0
```

**Review by mdenker — APPROVED** [review]

File: doc/conf.py (hunk 1 above)
Comment (Moritz-Alexander-Kern) [reply doc/conf.py:22]: thanks, added with eb9ff3804bfbff6ecc4bdaf923d04b114fdfef61

File: requirements/requirements-docs.txt (hunk 2 above)
Comment (Moritz-Alexander-Kern) [reply requirements/requirements-docs.txt]: Done: 98643fde05ce270407c339c66b52ff5422ac68a3

**Pull request merged**